pyproject.toml

```diff
@@ -19,85 +19,102 @@ packages = [{ include = "pyquil" }]
 exclude = ["pyquil/conftest.py"]
 
 [tool.poetry.dependencies]
-python = "^3.8,<=3.12"
-numpy = "^1.22"
-scipy = "^1.7.3"
-lark = "^0.11.1"
-rpcq = "^3.10.0"
+python = "^3.9,<=3.13"
```

Comment:
Does this include all the patch versions of `3.13`? See comment at https://github.com/quantumlib/Cirq/pull/6281#pullrequestreview-2094830342